### ISSUE TITLE
Font issue

### ISSUE THREAD

**Issue by karek314** [dashpay/dash#873]
<img width="43" alt="screen shot 2016-06-08 at 23 10 45" src="https://cloud.githubusercontent.com/assets/9548618/15911895/027efa10-2dd2-11e6-89db-04f586fcfcb4.png">
Issue in font.
OS: OSX Yosemite
Dash Core (0.12.0.57)

**Comment by UdjinM6** [comment]
Wow. What language?

**Comment by karek314** [comment]
English

**Comment by schinzelh** [comment]
What was the original value?

`-2.02 DASH`?

**Comment by karek314** [comment]
-2002.xxxxx Dash

**Comment by karek314** [comment]
Yes, but in other places "." dot is displayed correctly, just randomly any character get get corrupted (in transaction values)

**Comment by schinzelh** [comment]
That's strange :)

**Comment by nmarley** [comment]
Hi Karek, is this still an issue using the latest version of the software?

**Comment by karek314** [comment]
Hi @nmarley Sorry i don't have ability to test it right now.
